presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -428,29 +428,27 @@ public TaskManagerConfig setInfoMaxAge(Duration infoMaxAge)
     }
 
     @Min(1)
-    @PowerOfTwo
     public int getWriterCount()
     {
         return writerCount;
     }
 
     @Config("task.writer-count")
-    @ConfigDescription("Number of writers per task")
+    @ConfigDescription("Number of writer threads per worker")
```

Comment:
i would leave this description as per-task, as this is the TaskManagerConfig, and the control is in the task. (it just is also true that there is also one task per worker).

Reply:
Updated.